sync/02-cleanup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+. "$(dirname "${BASH_SOURCE[0]}")/../.common.sh"
+. "$(dirname "${BASH_SOURCE[0]}")/../.client.sh"
+
+log_message "02-cleanup.sh..."
+
+delete_managed "user"
```

Comment:
I am not a fan of this type of cleanup. Would be better if the test uses it's own data and maybe cleans it up during init phase.

Reply:
Cleanup has been completely removed.